packages/client/src/i18n/messages/views/action.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+import { defineMessages, MessageDescriptor } from 'react-intl'
+
+interface IActionMessages
+  extends Record<string | number | symbol, MessageDescriptor> {
+  action: MessageDescriptor
+  assignedTo: MessageDescriptor
+  view: MessageDescriptor
+  correctRecord: MessageDescriptor
+  archiveRecord: MessageDescriptor
+  reinstateRecord: MessageDescriptor
+  reviewCorrection: MessageDescriptor
+  reviewDeclaration: MessageDescriptor
+  updateDeclaration: MessageDescriptor
+  printDeclaration: MessageDescriptor
+  issueCertificate: MessageDescriptor
+}
```

Comment:
Do we need to define the types here actually or could we just directly pass the object as-is to `defineMessages`. Could help maintaining these

Reply:
Yea, I've occasionally removed these as they in reality make the typing worse :/